src/hooks/series/useSeriesData.ts

```diff
@@ -1,33 +1,33 @@
 import usePlaylist from '#src/hooks/usePlaylist';
-import useSeries from '#src/hooks/useSeries';
-import type { Playlist } from '#types/playlist';
+import useSeries from '#src/hooks/series/useSeries';
+import type { Playlist, PlaylistItem } from '#types/playlist';
 import type { Series } from '#types/series';
 
 const DEFAULT_DATA = { title: '', playlist: [] };
 
 type Data = {
-  seriesPlaylist: Playlist;
-  series: Series | undefined;
+  playlist: Playlist;
+  newSeries: { series: Series | undefined; media: PlaylistItem | undefined };
 };
 
 export const useSeriesData = (
-  entityId?: string,
+  seriesId: string | undefined,
+  mediaid: string | undefined,
```

Comment:
The variable naming here is hard to grasp. mediaid is a series ID and seriesID is a playlist ID. Suggest the following tweak:
```suggestion
  legacySeriesPlaylistId: string | undefined,
  seriesId: string | undefined,
```

Reply:
Good suggestion, thank you!